Makefile

```diff
@@ -31,7 +31,19 @@ dev-cluster: discovery-url
 	deisctl install platform
 
 discovery-url:
-	sed -e "s,discovery #DISCOVERY_URL,discovery $$(curl -s -w '\n' https://discovery.etcd.io/new?size=$$DEIS_NUM_INSTANCES)," contrib/coreos/user-data.example > contrib/coreos/user-data
+	@for i in 1 2 3 4 5; do \
```

Comment:
We discussed that but left it this way. It seemed like "seq 5" could imply everything is nicely parameterized and that you could change "5" to "10" and get ten retries, but since we had to hard-code "5" a few lines below, it actually seemed safer to hard-code all the values.